kasa/smartdevice.py

```diff
@@ -394,7 +417,7 @@ class itself as @requires_update will be affected for other properties.
         """
         return self._sys_info  # type: ignore
 
-    @property  # type: ignore
```

Comment:
It would be nicer if all typing fixes that are not directly related to introducing the `DeviceConfig` would be done in a separate PR.

Reply:
Removed